timesketch/frontend-ng/src/components/Explore/AddManualEvent.vue

```diff
@@ -75,12 +75,13 @@ limitations under the License.
       </v-row>
 
       <v-card-actions>
-        <v-spacer></v-spacer>
         <v-btn text color="primary" @click="attributes.push({ name: '', value: '' })" :disabled="isDisabled">
+          <v-icon>mdi-plus</v-icon>
           Add Attribute
         </v-btn>
-        <v-btn text color="primary" @click="clearAndCancel"> Cancel </v-btn>
-        <v-btn text color="primary" @click="submit"> Add Event </v-btn>
+        <v-spacer></v-spacer>
+        <v-btn text @click="clearAndCancel()"> Cancel </v-btn>
+        <v-btn text color="primary" @click="submit"> Save </v-btn>
```

Comment:
Three UX nits:
1. Hitting the escape key will close the dialog but not clear it. 
2. Hitting the enter key does not submit the form.
3. There are no errors or info what is missing or not when trying to submit a form with a field not entered. At least it should show which fields are required and missing.

Reply:
This form is complex, and need better validation before we can fix this. Will happen in a followup PR.